github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -1906,3 +1906,36 @@ presubmits:
           privileged: true
       nodeSelector:
         type: bare-metal-external
+  - always_run: true
```

Comment:
This needs to be set to `false` if `run_if_changed` is set.

Reply:
Done